src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParser.NetStandard.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Data.SqlClient
+{
+    internal sealed partial class TdsParser
+    {
+		internal static void GetGuidBytes(Guid guid, Span<byte> buffer)
```

Comment:
Methods prefixed with `Get` seem like they should return something. 
Can these be renamed to `FillGuidBytes` `FillDoubleBytes` etc?

Reply:
I wasn't sure about Get, I tried Write but that was an even worse fit. Fill is much better, changed.

Reply:
Thanks !